phys/module_sf_noahlsm.F

```diff
@@ -29,7 +29,7 @@ MODULE module_sf_noahlsm
 ! VEGETATION PARAMETERS
         INTEGER :: LUCATS , BARE
         INTEGER :: NATURAL
-        INTEGER :: LOW_DENSITY_RESIDENTIAL, HIGH_DENSITY_RESIDENTIAL, HIGH_INTENSITY_INDUSTRIAL
+        INTEGER :: LCZ_1,LCZ_2,LCZ_3,LCZ_4,LCZ_5,LCZ_6,LCZ_7,LCZ_8,LCZ_9,LCZ_10,LCZ_11
```

Comment:
If this line is removed, are there any consequences?

Reply:
No, it simply substitute all the old urban classes with the new ones